docs/modules/ROOT/pages/explanations/decisions/api-design.adoc

```diff
@@ -387,6 +387,11 @@ apiVersion: acmecorp.appcat.vshn.io/v1
 kind: AcmecorpPostgreSQL
 ----
 
+=== Addendum 1 Exposing Versions
+
+We expose the versioning of the various services via the composite as an enum.
+The default of the field should always point to the most recent stable version.
```

Comment:
What if we skip default altogether and leave exoscale to decide the latest version?

Reply:
The decision here isn't specific to Exoscale, but to all our services.